cli/azd/pkg/tools/git/git.go

```diff
@@ -143,6 +146,29 @@ func (cli *gitCli) InitRepo(ctx context.Context, repositoryPath string) error {
 	return nil
 }
 
+func (cli *gitCli) CheckConfigCredentialHelper(ctx context.Context) (bool, error) {
+	found, err := tools.ToolInPath("git")
```

Comment:
Why does this command run an extra check, while others dont?

Reply:
https://github.com/Azure/azure-dev/issues/756